src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -160,7 +160,38 @@ private void testEditSessionAction() throws Exception {
         feedbackEditPage.reloadPage();
         feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditSuccess.html");
 
-
+        ______TS("test two 'change' links expand uncommon settings panels");
+        
+        // click uncommon settings 'send emails'
+        feedbackEditPage.clickEditUncommonSettingsSendEmailsButton();
```

Comment:
If a comment applies to the next line only (i.e. not for the entire code block) either give it as a trailing comment,

```java
feedbackEditPage.clickEditUncommonSettingsSendEmailsButton(); //comment
```

or add a blank line to separate the line from the remaining code.

```java
// click uncommon settings 'send emails'
feedbackEditPage.clickEditUncommonSettingsSendEmailsButton();

feedbackEditPage.verifyEditSessionBoxIsEnabled();
```